### PR TITLE
Added Applications of MUX and DEMUX 

### LINKED ISSUE #32: Add applications of MUX and DEMUX
#### I am submitting a -
* [] **Feature Request**

#### Describe your bug/feature - 
The application of multiplexer and demultiplexer can be added to make the book more informative.

### PR DESCRIPTION
**Pull request by Prerna-0202**
Fixes #32 

I have added the applications of multiplexer and demultiplexer to make the book more informative.

#### Screenshots
Screenshot (5)
Screenshot (6)

#### ✅️ By submitting this PR, I have verified the following

- [ ] Checked to see if a similar PR has already been opened 🤔️
- [ ] Reviewed the contributing guidelines 🔍️
- [ ] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [ ] Tried Squashing the commits into one

### CODE REVIEW

**Comment by Prerna-0202** [comment]
@manjotsidhu, Can you please review my pr once and provide me with the reviews...

**Review by manjotsidhu — CHANGES_REQUESTED** [review]

**Comment by Prerna-0202** [comment]
@manjotsidhu, I have commit the changes. Can you please review it again...

File: docs/comb-msi/mux-demux.md [hunk 1]
```diff
@@ -124,6 +124,13 @@ It can be implemented with **two** 8 : 1 multiplexers:
 It can also be implemented with **five** 4 : 1 multiplexers:
 <iframe width="100%" height="400px" src="https://circuitverse.org/simulator/embed/44804" id="mux_06" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
 
+## Applications of Multiplexers
+- Transmission from the Computer System of a Satellite - Multiplexer is used to transmit the data signals from the computer system of a spacecraft or a satellite to the ground system by using a GSM satellite.
+
+• Computer Memory - Multiplexers are used in computer memory to maintain a huge amount of memory in the computers, and also to reduce the number of copper lines required to connect the memory to other parts of the computer.
```
Comment (manjotsidhu) [inline docs/comb-msi/mux-demux.md:130]: Here as well

Comment (manjotsidhu) [reply docs/comb-msi/mux-demux.md:130]: Make changes everrywhere

Comment (Prerna-0202) [reply docs/comb-msi/mux-demux.md]: @manjotsidhu, Done sir!!!

**Review by manjotsidhu — APPROVED** [review]

**Pull request merged**